### DISCUSSION TITLE
ITVx

### DISCUSSION THREAD

**Discussion by PC-Thai** [Ascoware/get-iplayer-automator#425] [General]
ITVx launches on 8th December and replaces the ITVHub. Are you planning to update GIA to reflect this? I just looked at the new website but it’s just advertising stuff at the moment …

**Comment by skovatch** [comment]
This is the first I'm hearing about it... I suspect I will have to update the code that scrapes the ITV listings, and I would guess there will be some work for just getting URLs for downloads.

**Comment by skovatch** [comment]
It's also worth noting that ITV downloads rely on yt-dlp, so that project will need to update first before I can do anything.

**Comment by PC-Thai** [comment]
Looks like ITV have implemented the changes already and GIA no longer seems to be updating the ITV Indexes. BBC is fine and working aok though!

**Comment by SOLOSAIR** [comment]
GiA now gets completely stuck trying to update ITV indexes.  This means I've had to uncheck 'ITV' box in settings in order to progress the app to listing/downloading the BBC programmes. It seems ITVx may have caused this.

**Comment by skovatch** [comment]
Well, that was fast. I can't even use 'get current webpage', so everything is basically broken for ITV. 

I am in day 4 of the new job and need to be on site, so don't expect any updates until this weekend at the earliest.

**Comment by WilliamOfGaunt** [comment]
Congratulations on the new job, Scott. I’m sure all on this list wish you well with it and are content to wait until you have time to fix the problem with ITV downloads. Personally I’m just grateful to you for all the work you’ve put into the project on your own time.

Bruce
